src/test-covariance.cpp

```diff
@@ -91,6 +91,38 @@ context("autoregressive") {
   }
 }
 
+context("compound symmetry") {
+  test_that("corr_fun_compound_symmetry works as expected") {
+    vector<double> theta {{1.0}};
+    corr_fun_compound_symmetry<double> test_fun(theta);
+    expect_equal(test_fun(1, 0), 1 / sqrt(2));
+    expect_equal(test_fun(4, 1), 1 / sqrt(2));
+    expect_equal(test_fun(3, 1), 1 / sqrt(2));
```

Comment:
Using a theta of 1 doesn't test for whether the correlation parameter is squared (eg, with `theta == 1`, `theta / sqrt(1 + theta * theta)` == `theta / sqrt(1 + theta)`. 

Probably unlikely to be problematic, but it would be more thorough to use something other than the multiplicative identity for this.

Reply:
Good idea, thanks, done